### ISSUE TITLE
Lucky eggs are not being used, evolve doesn't work. 

### ISSUE THREAD

**Issue by JulioMansilla** [PokemonGoF/PokemonGo-Bot#1335]
### Expected Behavior

"use_lucky_egg": true, will use lucky egg
"evolve_all": "all", will evolve all pokemons
### Actual Behavior

Lucky egg is not sued
Evolution process doesn't start
### Steps to Reproduce

set config file to:
"use_lucky_egg": true,
"evolve_all": "all",

load bot, no egg will be used, no pokemons will be evolved.
### Other Information
OS: Mac
Git Commit: 4c46ad7e6b41f4d66400aee82be469cdbbb26f80

**Comment by leadboots5** [comment]
I have the same experience. I will also note that "evolve_all" doesn't attempt to do any evolving on script start up.
15e0161d32c7a78c230c6e75cfd6867663c985d1

**Comment by JulioMansilla** [comment]
@leadboots5, I have the same issue with evolving too.

**Comment by leovarmak** [comment]
@JulioMansilla Same issue here too. The lucky egg is not being used

**Comment by beedy01** [comment]
evolve doesnt run properly, so am i, i got that issue..

**Comment by JulioMansilla** [comment]
I've edited the issue to reflect the evolve issue too.

**Comment by Vvkmnn** [comment]
I've also got this issue on e25a7091f5f26eef17bc38f6f90df26ad199ffbc but there was a PR to possibly solve it: #1256

It's still waiting on some testing before it gets merged.

**Comment by TheGoldenXY** [comment]
This issue has already been fixed.

Please upgrade to the latest `dev` branch by running this command:
```
git reset --hard
git pull -f
```

Then run this command:
`pip install --upgrade -r requirements.txt`

If issue persists, open a new issue.

/closed
